.github/workflows/ampd-handlers-docker.yaml

```diff
@@ -0,0 +1,168 @@
+name: ampd-handlers - Build and push images to ECR
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'ampd-handlers/**'
+      - 'packages/ampd-sdk/**'
+      - '.github/workflows/ampd-handlers-docker.yaml'
+  workflow_dispatch:
+    inputs:
+      handler:
+        description: 'Handler to build (leave empty for all)'
+        required: false
+        type: choice
+        options:
+          - '' # All handlers
+          - evm # List to be updated with all handlers
+        default: ''
+      ref:
+        description: 'Git ref (branch/commit/tag) to build from'
+        required: false
+        default: 'main'
+      tag:
+        description: 'Custom tag (leave empty to use git sha or tag name)'
+        required: false
+        default: ''
```

Comment:
if I understand correctly, this is the tag for the image that gets produced, not the tag of the code version that is checked out. What's the benefit of allowing custom tags at all?